source/Files.h

```diff
@@ -72,6 +72,7 @@ class Files {
 
 	// Open this user's plugins directory in their native file explorer.
 	static void OpenUserPluginFolder();
+	static void OpenUserSavesFolder();
```

Comment:
should probably add a comment here to match the `OpenUserPluginFolder()` comment